schedule/requirements.txt

```diff
@@ -3,7 +3,7 @@ argparse==1.1
 django-coverage==1.2.4
 django-nose==1.1
 nose==1.3.0
-python-dateutil==2.1
+altschool-python-dateutil==2.1
```

Comment:
I'll try that; this project uses the old setuptools in the root directory and does not have a requirements.txt file there.. not sure how this file is actually used